contents/docs/integrate/_snippets/install-js-snippet.mdx

```diff
@@ -1,19 +1,16 @@
 import Snippet from "../snippet.mdx"
 
-> This is the simplest way to get PostHog up and running on your website, and only takes a few minutes to set-up.
+This is the simplest way to get PostHog up and running, and only takes a few minutes.
 
-#### Add to your website &amp; app
+#### Add to your website and product
 
-Paste this snippet within the `<head>` tags of your website - ideally just inside the closing `</head>` tag - on all pages that you wish to track.
+Copy the snippet below and replace `<ph_project_api_key>` and `<ph_instance_address>` with your project's values, then add it within the `<head>` tags of your product - ideally just before the closing `</head>` tag - on pages you want to track. 
+
+You can find the snippet pre-filled with this data in [your project settings](https://app.posthog.com/project/settings).
```

Comment:
```suggestion
You can find the snippet pre-filled with this data in [your project settings](https://us.posthog.com/settings/project#snippet).
```

Reply:
Is US the right domain? I'm not sure if that will redirect to EU for relevant users

Reply:
The US domain redirects 😄